src/main.py

```diff
@@ -118,8 +121,9 @@ def run(self):
             print error_msg
             return
 
-        r = praw.Reddit('OAuth Baseball-GDT-Bot V. 3.0.1'
-                        'https://github.com/mattabullock/Baseball-GDT-Bot')
+        r = praw.Reddit('OAuth Baseball-GDT-Bot V. 3.0.1 '
+                        'https://github.com/mattabullock/Baseball-GDT-Bot '
+                        'Forked by /u/toddrob')

```

Comment:
I tried to remove this line in the pull request but either I did it wrong or it can't be done. I didn't mean for this to be merged into master.